### PR TITLE
Add Signature subclass for Transaction Signatures

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: test/transaction/signature.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    });
+
+    it('can roundtrip to/from json', function() {
+      var signature = getSignatureFromTransaction();
+      var serialized = signature.toObject();
+      var json = signature.toJSON();
+      expect(TransactionSignature(json).toObject()).to.deep.equal(serialized);
+      expect(TransactionSignature.fromJSON(json).toObject()).to.deep.equal(serialized);
+    });
+  });
```
Comment (maraoz) [inline test/transaction/signature.js]: please add a test deserializing from a JSON directly

**Comment by maraoz** [comment]
ACK

File: test/transaction/signature.js (hunk 1 above)
Comment (eordano) [inline test/transaction/signature.js]: done

**Comment by maraoz** [comment]
Coverage decreased (-45.71%)

**Pull request merged**